content/index.md

```diff
@@ -45,44 +28,94 @@ develop both our server-side code and our client-side code using components.
     <div class="card-body">
         <p>lorem ipsum dolor sit amet</p>
     </div>
-
-    <div class="card-controls">
-        <button type="button">toggle</button>
-    <div>
 </article>
 ```
 
 </div>
 </div>
 
-The complate approach is similar to server-side rendering in component-based
-frameworks like [React](https://reactjs.org/) and
-[Vue](https://vuejs.org/), but because their implementations have to deal with
-client-side rendering as well, they are larger and more difficult to port to
-other platforms outside of the JavaScript ecosystem.
-
-This is where complate shines: Because of the
-[lean underlying implementation](https://www.innoq.com/en/blog/self-contained-custom-elements/), it is possible to write a component
-which can easily be reused across multiple projects on different platforms.
-
-Cross-Platform Support
-----------------------
-
-With complate, components can be used across multiple platforms.
-[On the server](https://github.com/complate/complate-stream), we can integrate
-complate macros in different tech stacks like:
-
-* [Express](https://github.com/complate/complate-express) (Node.js)
-* [Ruby on Rails](https://github.com/complate/complate-ruby) (Ruby)
-* [Spring](https://github.com/complate/complate-spring) (Java)
-* etc.
-
-This is particularly useful in heterogeneous software systems, where we may be
-running multiple services using different back-end technologies, but we want to
-reuse front-end components between services. If this is the case, we can
-develop a [pattern library](https://github.com/complate/complate-fractal) using
-complate and then publish our macros for reuse in other services.
-
-We can also use complate
-[on the client](https://github.com/complate/complate-dom) and as a
-[static-site generator](https://github.com/complate/complate-ssg).
+This `Card` component is [implemented as a JavaScript function](#components).
+
+
+Stateless HTML Rendering
+------------------------
+
+complate focuses on efficiently [generating HTML],
+with support for [progressive rendering](#progressive-rendering).
+This makes this templating language ideal for [server-side rendering],
```

Comment:
minor nit: "This makes this" is awkward duplication - perhaps "This makes complate a templating language ideal for server-side rendering" or "Thus it's ideal for server-side rendering"? No big deal either way though.